### ISSUE TITLE
Please update docker for image on docker hub

### ISSUE THREAD

**Issue by kostakoff** [jenkinsci/jenkinsfile-runner#720]
Please update docker image on docker hub https://hub.docker.com/r/jenkins/jenkinsfile-runner to version 1.0-beta-32

**Comment by hfoffani** [comment]
Yes, please.
On dependency of the docker plugin requires Jenkins 2.440.1 for instance.

**Comment by tiago-teixeira5** [comment]
Hi,
I believe they now use the [github repository](https://github.com/jenkinsci/jenkinsfile-runner/pkgs/container/jenkinsfile-runner), which seem to have the recent versions.

**Comment by oleg-nenashev** [comment]
Yes. As of now, no plans to maintain the DockerHub image until 1.0.
But maybe something to revisit
